stately-async/src/reducer.ts

```diff
@@ -37,19 +37,24 @@ const errorReducer: Reducer<AsyncState<any, any>, Action> = (state = initialAsyn
     : state
 
 const completeReducer: Reducer<AsyncState<any, any>, Action> = (state = initialAsyncState, action) =>
-  isAsyncAction(action) && action[StatelyAsyncSymbol].phase === 'complete'
-    ? {
-        ...state,
-        status: 'completed',
-        error: null,
-      }
+  isAsyncAction(action) && action[StatelyAsyncSymbol].phase === 'reset'
```

Comment:
huh?

Reply:
yeah holdup something got mixed up
the `completeReducer` handles `reset` and the `resetReducer` handles `complete`